src/AirshipBindings.Android.ADM/AirshipBindings.Android.ADM.csproj

```diff
@@ -90,7 +90,7 @@
     <Folder Include="Jars\" />
   </ItemGroup>
   <ItemGroup>
-    <LibraryProjectZip Include="Jars\urbanairship-adm-9.2.0.aar" />
+    <LibraryProjectZip Include="Jars\urbanairship-adm-9.4.0.aar" />
```

Comment:
I'm betting we can figure out a way to reference `airship.properties` or some other file so we don't have to hardcode the versions in the csproj files.

Reply:
I think that's what happened here. I didn't change the reference, just updated airship.properties and ran the build script.

Reply:
I forgot about that. That's probably good enough.